### DISCUSSION TITLE
SMB Mount at Setup

### DISCUSSION THREAD

**Discussion by ghost** [tteck/Proxmox#1865] [Ideas]
Add options to those lxc scripts that may need them, to configure any fstab smb shares during the setup eg. Plex, Jellyfin, qBittorrent, Sonarr, Radarr etc.

**Comment by fodi666** [comment]
Yes, that would be a very good point, config folders or media folders could use SMB mounts that could be added during setup

**Comment by AlexGeek77** [comment]
I've been wrestling with the best way to accomplish this task and my findings indicate it would be too complicate to script successfully. There are just too many ways to accomplish this as well as exactly how you tackle the UID/GID mapping. 

For me, I've got all my media on a NAS that has a SMB share which has specific users (sonarr, radarr, sabnzbd) who all have access. So I mount the SMB (with the specific credentials) for each onto the Proxmox host which gives me lots of options like /mnt/pve/sonarr-video and /mnt/pve/sabnzbd-download and /mnt/pve/radar-video. I create a local user on the proxmox host for each LXC and map the UID to the newly created local accounts. Give those accounts ownership of the mounted CIFS shares. 

It's been a bit of a pain honestly and has taken way too long to finally get things going. I've finally got sonarr up and going from a backup of my current install as I make sure I can properly port it over to Proxmox.
